GetCertificate/handler.ts

```diff
@@ -82,8 +82,10 @@ export const processSuccessCertificateGeneration = (
       i => i.status === 200,
       () => ({
         header_info: getFallbackHeaderInfoForLanguage(selectedLanguage),
-        info: printExpiredInfo(selectedLanguage) || EMPTY_STRING_FOR_MARKDOWN,
-        status: ExpiredEnum.expired
+        info:
+          printExpiredOrRevokedInfo(selectedLanguage) ||
+          EMPTY_STRING_FOR_MARKDOWN,
+        status: RevokedEnum.revoked
```

Comment:
I could not cope with all previous conversations so I may have lost some bits context. 

By doing so we're treating every 404 as revoked, which is an hard statement. That include tests certificates which normally expire after 48hrs.

That's fine for me if you already considered all the scenarios, I just wanted to double check this decision.

Reply:
yes, we took this into account and decided that revoked status (and its description) is more generic.
But thanks for the input, I should have wrote it into PR description.